test/util/wdc-interface.test.coffee

```diff
@@ -13,7 +13,7 @@ describe 'Web Data Connector Interface', ->
     checkResult = () ->
```

Comment:
If the above is an error, the test should then be modified to check for positional elements `> 10`.